python/nano/src/bigdl/nano/common/cpu_schedule.py

```diff
@@ -30,17 +31,20 @@ def get_cgroup_cpuset():
         cpu_set = [i for i in range(int(subprocess.check_output(
             get_physical_core_args, universal_newlines=True).splitlines()[4].split("=")[1]))]
     else:
-        with open("/sys/fs/cgroup/cpuset/cpuset.cpus", "r") as f:
-            content = f.readlines()
-        cpu_set = []
-        values = content[0].strip().split(",")
-        for value in values:
-            if "-" in value:
-                # Parse the value like "2-4"
-                start, end = value.split("-")
-                cpu_set.extend([i for i in range(int(start), int(end) + 1)])
-            else:
-                cpu_set.append(int(value))
+        try:
+            with open("/sys/fs/cgroup/cpuset/cpuset.cpus", "r") as f:
+                content = f.readlines()
```

Comment:
we may use `pycpuinfo`?

Reply:
Actually, I'm not sure whether to use `cpuinfo`, let me explain why.
1. Compared to `os`, `cpuinfo` is too slow, 
```python
%time cpuinfo.get_cpu_info()["count"]
# CPU times: user 3.03 ms, sys: 4.17 ms, total: 7.2 ms
# Wall time: 1.21 s
20

%time os.cpu_count()
CPU times: user 55 µs, sys: 5 µs, total: 60 µs
Wall time: 65.3 µs
20
```
2. Because I still have a lot of questions, the `cpuset.cpu` records seem to be for physical cores rather than logical cores?
related link: https://man7.org/linux/man-pages/man7/cpuset.7.html
btw, I tried this code on my personal server(windows 6c12t), the number of cores obtained is 6, not 12.

Reply:
> we may use `pycpuinfo`?

Compared to `os`, `pycpuinfo` is too slow, 
```python
%time cpuinfo.get_cpu_info()["count"]
# CPU times: user 3.03 ms, sys: 4.17 ms, total: 7.2 ms
# Wall time: 1.21 s
20

%time os.cpu_count()
CPU times: user 55 µs, sys: 5 µs, total: 60 µs
Wall time: 65.3 µs
20
```